packages/story-editor/src/components/library/panes/shopping/productImage.js

```diff
@@ -32,11 +37,24 @@ const StyledImgPlaceHolder = styled.div`
   ${imgPlaceholder}
 `;
 
+const ProductImageWrap = styled.div`
+  position: relative;
+`;
+
 const StyledImage = styled.img`
   ${imgPlaceholder}
   object-fit: cover;
 `;
 
+const StyledImageClone = styled.img`
```

Comment:
Maybe it would make sense to instead of showing the image, showing the actual CTA button that's displayed on the page?
This would:
- Display the user what will actually be added on the page
- Fix snapping while dragging (the other elements snap to the dragged image, however, that's not actually added so it isn't very useful)
- Fix being able to drag a product outside of the page. Generally, when an element is dragged outside of the page, it is deleted, or not added - in case of dragging a new element from the library. However, this is calculated based on the element that's being dragged. If we're displaying the image instead of the button that's actually added to the page, it's possible to drag a product so that it's created a bit outside of the page, for example:
![dragging](https://user-images.githubusercontent.com/3294597/179529612-9f503fe0-d024-43c1-a8db-681b0d1926cc.gif)

Reply:
@swissspidy let me know how you think we should proceed based on these comments.

Reply:
(One alternative would be displaying the image but 25x25 instead)

Reply:
Using a 25x25 image should be fine for now. We can revisit later if needed